modules/aruco/src/apriltag/apriltag_quad_thresh.cpp

```diff
@@ -1567,4 +1567,121 @@ imwrite("2.5 debug_lines.pnm", out);
     return quads;
 }
 
+void _apriltag(Mat im_orig, const Ptr<DetectorParameters> & _params, std::vector< std::vector< Point2f > > &candidates,
+        std::vector< std::vector< Point > > &contours){
+
+    ///////////////////////////////////////////////////////////
+    /// Step 1. Detect quads according to requested image decimation
+    /// and blurring parameters.
+    Mat quad_im;
+    im_orig.copyTo(quad_im);
```

Comment:
Most probably it should be in else branch of the next condition to not do extra memory copy for the resize case.

Reply:
fixed